draft-ietf-rtcweb-jsep.xml

```diff
@@ -1595,7 +1595,27 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               "max-bundle", and this is not the first m= section, or the BUNDLE
               policy is set to "balanced", and this is not the first m= section
               for this media type, an "a=bundle-only" line.</t>
-            </list></t>
+
+	      <t>If the RtpTransceiver's RtpSender is active, an "a=rid" line
+	      for each encoding specified by the application in the RtpSenders's
+	      parameters.  If no encodings have been specified, then one RID
+	      line is generated.  The rid line is specified in <xref
+	      target="I-D.ietf-mmusic-rid"></xref> and indicates the RID value
```

Comment:
Do we need two references in the same sentence? Perhaps we want to refer to separate sections?